app/presenters/workshop_presenter.rb

```diff
@@ -96,7 +96,9 @@ def pairing_csv
   private
 
   def students_checklist
-    model.attending_students.order('note asc').each_with_index.map do |a, pos|
+    # TODO: review if it needs this removed scope or not
+    #       Not going to worry unless the rest of the code is OKd
```

Comment:
TODO can be removed as this does not require `not_banned` scope to be used. Ideally members should be removed from all RSVPed events as soon as they are banned, but even if this is missed, organisers would be able to see they are banned and they wouldnt be able to RSVP to any other future events so that is sufficient.

Reply:
`attending_students` along with all the other Invitable methods will remove banned Members. 

The TODO was to remind me that `.order('note asc')` was breaking the code. I don't actually know how it works now because there is already an ordering on attending_students [`.order('members.name, members.surname')`]  so a second in the chain order is ignored anyway. 

```
# current situation before I did anything was in effect:
.order('members.name, members.surname').order('note asc')
```


I will remove `order('note asc') `along with the TODO.

Reply:
I saw the PR on Slack and completely missed that it was WIP when I added my comments last night.

If anything, I think it would be more beneficial to order the attendees by RSVP time so that it gives a better idea to organisers about when someone RSVPed without having to necessarily check all logged RSVP times.

Reply:
It sounds like slack gives you the PR name and not the Github title - not sure what to do about that. 

It's definitely fixable. The ordering could be removed from the invitable#attending_students and invitable#attending_coaches and the order explicitly applied by having a `by_name` `by_rsvp_time` It looks like it would affect about 10 places. 🤷‍♂️

Reply:
@notapatch ah ok. Sounds like something for a new PR then so let's ignore my comment about changing the ordering here and I'll open a new issue.

Reply:
https://github.com/codebar/planner/issues/1275